src/braket/ahs/shifting_field.py

```diff
@@ -94,15 +94,44 @@ def stitch(
         Args:
             other (ShiftingField): The second shifting field to be stitched with.
             boundary (StitchBoundaryCondition): {"mean", "left", "right"}. Boundary point handler.
+
                 Possible options are
-            * "mean" - take the average of the boundary value points of the first
-              and the second time series.
-            * "left" - use the last value from the left time series as the boundary point.
-            * "right" - use the first value from the right time series as the boundary
-              point.
+
+              - "mean" - take the average of the boundary value points of the first
```

Comment:
I think you may need to indent this to work properly.

Reply:
Same as below, indent by 4 causes the rendering issue of making half the sentence as bold.